src/slider/index.html

```diff
@@ -217,6 +217,90 @@
 			</tp-slider-nav>
 			<tp-slider-count current="1" total="4" format="$current / $total">1 / 4</tp-slider-count>
 		</tp-slider>
+
+		<br>
+
+		<!-- slider with 2 slide per view -->
+		<tp-slider slides-per-view="2" infinite="yes">
```

Comment:
@benerd We already have some sliders in here which have slides per view, can we just add this attribute there?